inference-engine/tests/functional/plugin/shared/include/behavior/exec_graph_info.hpp

```diff
@@ -146,7 +148,9 @@ TEST_P(ExecGraphTests, CheckExecGraphInfoAfterExecution) {
     // Create CNNNetwork from ngrpah::Function
     InferenceEngine::CNNNetwork cnnNet(function);
     InferenceEngine::CNNNetwork execGraph;
-    if (targetDevice != CommonTestUtils::DEVICE_MULTI && targetDevice != CommonTestUtils::DEVICE_GNA) {
+    if (targetDevice != CommonTestUtils::DEVICE_MULTI &&
+        targetDevice != "TEMPLATE" &&
```

Comment:
May be, we will add `CommonTestUtils::TEMPLATE` constant....

Reply:
Done

Reply:
Great!